airflow/providers/cohere/provider.yaml

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+package-name: apache-airflow-providers-cohere
+
+name: Cohere
+
+description: |
+    `Cohere <https://docs.cohere.com/docs>`__
+
+suspended: false
+
+versions:
+  - 1.0.0
+
+integrations:
+  - integration-name: Cohere
+    external-doc-url: https://docs.cohere.com/docs
+    tags: [software]
+
+dependencies:
+  - apache-airflow>=2.4.0
```

Comment:
```suggestion
  - apache-airflow>=2.5.0
```

Reply:
Why is this restriction?

Reply:
I believe we upgrade the airflow version in the provider based on airflow release and the current minimum airflow version is 2.5.0 https://github.com/apache/airflow/blob/main/PROVIDERS.rst#minimum-supported-version-of-airflow-for-community-managed-providers

Reply:
Changed it to `2.5.0`